<launch>
  <arg name="input" default="/camera/depth_registered/points" />
  <arg name="mask" default="false" />
  <arg name="image" default="/camera/rgb/image_rect_color" />
  <node pkg="point_cloud_image_creator" type="point_cloud_image_creator_node"
	name="point_cloud_image_creator" output="screen">
    <remap from="input_info" to="/camera/depth_registered/camera_info" />
    <remap from="input" to="$(arg input)" />
    <remap from="in_image" to="$(arg image)" />
    <param name="mask_image" value="$(arg mask)" type="bool" />
    <remap from="mask_image" to="/point_cloud_image_creator/mask_image" />
   
  </node>
  <!-- <node pkg="image_view" type="image_view" -->
  <!-- 	name="image_view" output="screen" > -->
  <!--   <remap from="image" to="/cloud_image/output/image" /> -->
  <!-- </node> -->
</launch>
